ifcb-parsing-processing/make_csv_from_hdr_parse/hdrFileSummaryFull.csv

```diff
@@ -0,0 +1,7 @@
+,Filename,FileComment,triggerCount,roiCount,runTime,inhibitTime,humidity,temperature,PMTAhighVoltage,PMTBhighVoltage,SyringeSampleVolume,syringeSamplingSpeed,RunFastFactor,sampleNumber,runSampleFast
+0,D20240326T151003_IFCB206,in-lab testing,1158,1211,241.01736111111111,96.31225694444444,31.886935225452056,21.219401083390558,0.39,0.68,5,20,5,1,True
+1,D20240326T151750_IFCB206,in-lab testing,568,538,241.05223958333335,46.98400173611111,27.118069733730074,21.219401083390558,0.39,0.68,5,20,5,2,True
+2,D20240326T152537_IFCB206,in-lab testing,560,504,241.02774305555556,46.32995659722222,26.79440451667049,21.325848020141926,0.39,0.68,5,20,5,3,True
+3,D20240326T153324_IFCB206,in-lab testing,464,415,241.06288194444446,38.34631076388889,26.723138780804156,21.43063172350653,0.39,0.68,5,20,5,4,True
+4,D20240326T154111_IFCB206,in-lab testing,426,370,241.03661458333335,35.38616319444444,25.867949950408182,21.537078660257883,0.39,0.68,5,20,5,5,True
+5,D20240326T202714_IFCB206,in-lab testing,581,568,1201.107638888889,48.20488715277778,26.99929350728619,21.538741893644634,0.39,0.68,5,20,5,2,False
```

Comment:
I would delete this from the repo since we have newer example output files from the notebooks that should be used, but this can be done after merging this PR.